bpfman-ns/src/main.rs

```diff
@@ -111,11 +111,7 @@ fn execute_uprobe_attach(args: UprobeArgs, bpfman_pid: u32) -> anyhow::Result<()
         .context("failed to get UProbe from pin file")?;
 
     // Set namespace to target namespace
-    set_ns(
-        target_mnt_file,
-        CloneFlags::CLONE_NEWNS,
-        args.container_pid as u32,
-    )?;
+    enter_mnt_ns(target_mnt_file, args.container_pid as u32)?;
```

Comment:
nit: This doesn't need to be renamed in this PR

Reply:
Removed.